norns/control-clock.md

```diff
@@ -119,6 +119,27 @@ An example entry:
 
 You'll notice that these entries are identical to the ones located on the mapping screen.
 
+## keyboard
```

Comment:
are there any other keyboard functions that exist that we missed? (already existed?)

might be worth mentioning that kb works for password entry/etc, or typing file names

Reply:
raaaad, yes yes!